README.md

```diff
@@ -17,14 +17,41 @@ Installs and configures PostgreSQL as a client or a server.
 
 ### Chef
 
-- Chef 12.1+
+- Chef 12.5+
```

Comment:
If we're doing custom resource it's actually 12.7. 12.5/12.6 had a pile of issues with action_class